sb_vision/tokens.py

```diff
@@ -196,7 +215,8 @@ def infer_location_from_homography_matrix(
         *,
         homography_matrix,
         distance_model,
-        image_size
+        image_size,
+        marker_size,
```

Comment:
Trailing comma here is invalid in Python 3.5 (raises `SyntaxError`). I'm not sure why this doesn't show up on the CI (fixed in 3f4cd37435745a5bd985dedc7235cd8c014d0966).